### PR TITLE
merge press and news pages

### PR DESCRIPTION
**Pull request by Dindihub**
This PR covers issue #15 

I would appreciate your input @chalin @lukpueh

### CODE REVIEW

File: content/en/resources/news.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-description:
-  The [press](resources/press) page contains a full listing of news coverage
+description: TUF news coverage
+# aliases: [/news/press]
 ---
 
+## Highlights
+
+Here are our news highlights. For a complete list, see the
+[Press](#press)section.
```
Comment (lukpueh) [inline content/en/resources/news.md]:
```suggestion
[Press](#press) section.
```

File: content/en/resources/news.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 ---
 title: News
-description:
-  The [press](resources/press) page contains a full listing of news coverage
+description: TUF news coverage
+# aliases: [/news/press]
```
Comment (lukpueh) [inline content/en/resources/news.md]: Can't remember the "front matter" syntax from the top of my head, but this does not seem to work. Does the `#`  comment it out?

Also, IIUC, @chalin asked to make both "/news" and "/press" an alias for this page ( see #15)

**Review by lukpueh — CHANGES_REQUESTED** [review]
Please fix the missing white space and check with @chalin about the aliases, otherwise, this looks great! :)

File: content/en/resources/news.md (hunk 2 above)
Comment (Dindihub) [reply content/en/resources/news.md]: > Can't remember the "front matter" syntax from the top of my head, but this does not seem to work. Does the `#` comment it out?
> 
> Also, IIUC, @chalin asked to make both "/news" and "/press" an alias for this page ( see #15)

Thanks for the review @lukpueh . I thought that was the Docsy syntax, but I'll wait for chalins input.

Comment (Dindihub) [reply content/en/resources/news.md]: > Can't remember the "front matter" syntax from the top of my head, but this does not seem to work. Does the `#` comment it out?
> 
> Also, IIUC, @chalin asked to make both "/news" and "/press" an alias for this page ( see #15)

@lukpueh so you mean have two alliases?

File: content/en/resources/news.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
 ---
 title: News
-description:
-  The [press](resources/press) page contains a full listing of news coverage
+description: TUF news coverage
+aliases: -[/news] -[/press]
```
Comment (lukpueh) [inline content/en/resources/news.md]: Please check the syntax reference for Hugo front matter: https://gohugo.io/content-management/front-matter/#aliases

**Review by lukpueh — CHANGES_REQUESTED** [review]

File: content/en/resources/news.md (hunk 3 above)
Comment (Dindihub) [reply content/en/resources/news.md]: > Please check the syntax reference for Hugo front matter: https://gohugo.io/content-management/front-matter/#aliases

@lukpueh I followed that convention but the format checks do not allow me to do a multiline on the front matter. I'll wait to hear from @chalin to see how to go about it. Thanks for the reference.

File: content/en/resources/publications.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Publications
-description:
+description: Publications on securing software updater systems
```
Comment (chalin) [inline content/en/resources/publications.md]: The `description` field is optional. I'd drop it:

```suggestion
```

File: content/en/resources/news.md (hunk 3 above)
Comment (chalin) [inline content/en/resources/news.md]: Almost there:

```suggestion
aliases: [/news, /press]
```

Comment (chalin) [reply content/en/resources/news.md]: I've fixed this. It's standard YAML syntax, which can be either an inline list or a one-entry-per-line list.

**Review by chalin — APPROVED** [review]
LGTM but I'd like for you to pull in the changes that I just merged first so that we can test the aliases. That is, the following should successfully redirect:

- https://deploy-preview-17--tufio.netlify.app/news
- https://deploy-preview-17--tufio.netlify.app/press

**Comment by Dindihub** [comment]
> LGTM but I'd like for you to pull in the changes that I just merged first so that we can test the aliases. That is, the following should successfully redirect:
> 
> * https://deploy-preview-17--tufio.netlify.app/news
> * https://deploy-preview-17--tufio.netlify.app/press

@chalin unfortunately, the aliases are not redirecting

**Comment by chalin** [comment]
> @chalin unfortunately, the aliases are not redirecting

You need to rebase in order for the aliases to start working.

**Comment by chalin** [comment]
I've rebased for you. If you'd like to try it on your own another time, the commands to use are something like this:

```sh
git switch main
git pull
git switch press-news-merge
git rebase main
git push -f
```

**Comment by chalin** [comment]
Aliases are working now, as expected. I believe that this addresses all previously raised concerns, so let's merge this now.

**Pull request merged**